vmm/src/device_manager.rs

```diff
@@ -3350,6 +3371,20 @@ impl DeviceManager {
     }
 }
 
+#[cfg(feature = "acpi")]
+fn numa_node_id_from_memory_zone_id(numa_nodes: &NumaNodes, memory_zone_id: &str) -> Option<u32> {
+    for (numa_node_id, numa_node) in numa_nodes.iter() {
```

Comment:
`.iter()` implements `.find()` so you might be able to simplify this.

Reply:
Well since `numa_nodes` is a `HashMap`, I guess `find()` would return `Option<(u32, NumaNode)>` instead of `Option<u32>`. It looks simpler to keep this as it is, don't you think?